pkg/auth/response/response.go

```diff
@@ -23,30 +23,34 @@ import (
 
 // AuthResponse is the unify way of returing a AuthInfo with AuthData to SDK
 type AuthResponse struct {
-	UserID      string                  `json:"user_id,omitempty"`
-	Profile     userprofile.UserProfile `json:"profile"`
-	Roles       []string                `json:"roles,omitempty"`
-	AccessToken string                  `json:"access_token,omitempty"`
-	LastLoginAt *time.Time              `json:"last_login_at,omitempty"`
-	LastSeenAt  *time.Time              `json:"last_seen_at,omitempty"`
-	Verified    bool                    `json:"verified"`
-	VerifyInfo  map[string]bool         `json:"verify_info"`
+	UserID      string           `json:"user_id,omitempty"`
+	Metadata    userprofile.Data `json:"metadata,omitempty"`
```

Comment:
It seems better to keep metadata empty dictionary instead of using `omitempty`. This would be easier for client side to decode the data, because it can always assume the key `metadata` there.